ydb/docs/en/core/reference/ydb-cli/export-import/_includes/tools_restore.md

```diff
@@ -87,3 +91,9 @@ Matching schemas between the database and file system:
 ```bash
 {{ ydb-cli }} -p quickstart tools restore -p dir1/dir2 -i ~/backup_quickstart --dry-run
 ```
+
+### Recommended options for better performance
+
+```bash
+{{ ydb-cli }} -p quickstart tools restore -p . -i . --import-data --bandwidth=1GiB
```

Comment:
Why does limited bandwidth improve performance?

Reply:
If the `--bandwidth` is specified, the `--rps` limit is not applied. The default `--rps` limit is 30, and it doesn't allow restore to be fast enough.

Changed the text accordingly.